src/components/operations/operation-details/react/runtime/OperationDetailsWebsocket.tsx

```diff
@@ -85,13 +92,22 @@ export const OperationDetailsWebsocket = ({
     }
 
     return (
-        <div className={"operation-details-container"}>
+        <div className={"operation-details-container"}>            
             <Subtitle1 block className={"operation-details-title"} id={"operation"}>Operation</Subtitle1>
             {working 
                 ? <Spinner label="Loading..." labelPosition="below" size="small" />
                 : !operation
                     ? <Body1>No operation selected.</Body1> 
                     : <div className={"operation-details-content"}>
+                        {/* <OperationConsole
+                            isOpen={isConsoleOpen}
+                            setIsOpen={setIsConsoleOpen}
+                            api={api}
+                            operation={operation}
+                            apiService={apiService}
+                            oauthService={oauthService}
+                            routeHelper={routeHelper}
+                        /> */}
```

Comment:
is this needed to be added?

Reply:
Yes, I will add it in the next iteration. For now I was asked to deploy what I have for testing.